dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -396,7 +409,8 @@ public Experiment addVariant(final String experimentId, final String variantDesc
                 .description(Optional.of(variantDescription)).build());
 
         final ExperimentVariant experimentVariant = ExperimentVariant.builder().id(variantName)
-                .description(variantDescription).weight(0).build();
+                .description(variantDescription).weight(0)
+                .url(page.getPageUrl()+"?variant="+variantName).build();
```

Comment:
the name of the QueryParams is varaintName

Reply:
Done